test/DebugInfo/DebugInfo-GV-with-DIE.ll

```diff
@@ -0,0 +1,24 @@
+;; Ensure that DIExpressions are preserved in DIGlobalVariableExpressions
+;; This utilizes SPIRV DebugGlobalVariable's optional field DIExpression
+
+; RUN: llvm-as %s -o %t.bc
+; RUN: llvm-spirv -o %t.spv %t.bc --spirv-debug-info-version=nonsemantic-shader-200
```

Comment:
like this we are not limited to nonsemantic-shader-200, but also include nonsemantic-shader-100

Reply:
Added checking for nonsemantic-shader-100